docs/tutorials/motor_noise_debugging.rst

```diff
@@ -194,3 +194,80 @@ vibration resistant features include, but are not limited to the following:
 
 Additionally, consider making redundant ground connections through a fuse or polymeric positive temperature coefficient device (PTC) to prevent high current flows through 
 a carbon fiber frame.
+
+Grounding Vetiq Motors Frames
+#####################################################
+
+Vertiq's Pro Kit motors as well as many modular motors in the 40, 60, and 81 diameter sizes are shipped with a user accessible, nonpopulated component footprint. 
```

Comment:
instead of sizes can you say "families" because that's the verbiage we use in the rest of the documentation